### PR TITLE
Update QUEST_en.tsv

### PR DESCRIPTION
**Pull request by Azumii**
16 Removed "to"
18 "it" added
20 removed "of"
26 "you" added
93 modified by "My duty is to"
247 modified Stary by Stay
469  modified "That guy playing witha furry friend" by "That guy who's playing with a furry friend"

### CODE REVIEW

File: QUEST_en.tsv [hunk 1]
```diff
@@ -13,17 +13,17 @@ QUEST_20150317_000012	Bokor Master
...
 QUEST_20150317_000014	Goddess Laima
 QUEST_20150317_000015	By the time you see this revelation, a long time must have passed by. {Nl} I am Laima, the Goddess of Fate and Luck. {Nl}
-QUEST_20150317_000016	When I made this revelation to 600 years ago, I saw a lot of grief and death. {Nl} Your seeing this revelation would mean that all I predicted have become reality. {Nl}
+QUEST_20150317_000016	When I made this revelation 600 years ago, I saw a lot of grief and death. {Nl} Your seeing this revelation would mean that all I predicted have become reality. {Nl}
 QUEST_20150317_000017	The first disaster was cause by the Demons, and all the Goddesses may have disappeared. {Nl} I know all of this beforehand and tried my best to prevent it from happening, but my power was not enough. {Nl}
-QUEST_20150317_000018	I walked through the long path of predictions to find the answer. {Nl} At the end of the dream was you.. The Savior. {Nl}
+QUEST_20150317_000018	I walked through the long path of predictions to find the answer. {Nl} At the end of the dream, it was you.. The Savior. {Nl}
```
Comment (emailboxu) [inline QUEST_en.tsv]: "dream was you" is correct. Read the sentence again. It implies that the Goddess was waiting at the end of the dream.

File: QUEST_en.tsv [hunk 2]
```diff
@@ -13,17 +13,17 @@ QUEST_20150317_000012	Bokor Master
...
+QUEST_20150317_000020	I decided to divide and put myself in multiple revelations. {Nl} Making and hiding myself in revelations around the kingdom continued for a thousand years. {Nl}
 QUEST_20150317_000021	Dear Savior. {Nl} In order for me to stop the Demons, {nl}you need to find all the revelations that I have hidden in the Axis of Time.
 QUEST_20150317_000022	The burden of the sacred mission could press down your shoulders. {nl} . {Nl}
 QUEST_20150317_000023	But, only you and I could save the world.{nl}
 QUEST_20150317_000024	I passed the next revelation to the first Paladin.{nl}I will be waiting for you at the garden at the top.
 QUEST_20150317_000025	Evil forces and will be but I do not feel at all. {Nl} Then, this revelation is true of Laima goddess .. just awesome.
-QUEST_20150317_000026	Okay, Please go back to Uska.{nl}He will tell you where to go.{nl}May the Goddess bless for your future..
+QUEST_20150317_000026	Okay, Please go back to Uska.{nl}He will tell you where to go.{nl}May the Goddess bless you for your future..
```
Comment (emailboxu) [inline QUEST_en.tsv]: It is more correct to remove "for" instead of adding "you" so it reads "May the Goddess bless your future."

Comment (megaoka) [inline QUEST_en.tsv]: :+1:

**Comment by imcgames** [comment]
Thank you :+1:

**Pull request merged**